build_envs/Darwin.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - jupyter
   - matplotlib
   - netcdf4
-  - numpy
+  - numpy!=1.24.3
```

Comment:
Normally I would agree, but the fix for this in numpy is part of the 1.24.4 roadmap and doing an exclude instead of creating a max version will allow us to not need to revisit this in a few weeks when 1.24.4 is released.

Reply:
I see; thanks for the clarification! Approving it.

Reply:
There is a [fix](https://github.com/numpy/numpy/pull/23680) that's been merged on numpy and slated for 1.24.4, but I haven't tested against that yet.  

Probably safer to do this though.  I'll push an update.

Reply:
Ah, shoot!  Just kidding - will leave it since it sounds like you all are ok with that.